Agilefreaks/weatherios
# Feature/select desired location

## Description

Use case
As a user I want to have a view so that I can search for new locations.

Acceptance criteria
I will be able to search for locations to see the weather info.

## Related

https://trello.com/c/l2yoZg1l

## Documentation

https://developer.apple.com/documentation/mapkit

## Link to demo

![Simulator Screen Shot - iPhone 12 - 2020-12-11 at 11 47 38](https://user-images.githubusercontent.com/15888842/101888325-ac85ec80-3ba6-11eb-8f59-4b05c086af1d.png)

## Steps to Test or Reproduce

1. Run app on device
2. Accept location permissions
3. You should see the home view with the current location weather data
4. You can press "Select City" button to switch to City Search View
5. Type desired city into search bar
6. Tap on desired result
7. Search view will disappear and the home view will be updated with the selected city weather data
